c2corg_ui/static/js/auth/auth.js

```diff
@@ -151,10 +161,19 @@ app.AuthController.prototype.successRegister_ = function(response) {
  * @private
  */
 app.AuthController.prototype.errorRegister_ = function(response) {
-  // TODO
-  alert('register error');
-  console.log('register error');
-  console.log(response);
+  // TODO: i18n
+  // FIXME: HTML is not interpreted in alert messages
+  var msg = 'Registration failed because of:';
+  var errors = response['data']['errors'],
+      len = errors.length;
+  if (len > 0) {
+    msg += '<ul>';
```

Comment:
> Have you considered doing the formatting in c2corg_ui/static/partials/alert.html instead?
Passing a list of submessages should avoid the sanitization issue.

I have tried to add HTML formating directly in the "alert" directive partial: it indeed works. The close button and the timeout closing too.
But as I said, it's not possible to know in advance what will be the content of the message (lists, links, bold/italic words, etc.).